src/Common/RoomClass.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace ATDev\RocketChat\Common;
+
+/**
+ * Room class
+ */
+class RoomClass extends Request
```

Comment:
As far as `Room` is a [separate](https://docs.rocket.chat/api/rest-api#rooms) resource I suggest to move `RoomClass` from `Common`, rename to `Room` and rename `Room` trait.